upstream/CVE-2021-25737.json

```diff
@@ -0,0 +1,113 @@
+{
+  "id": "CVE-2021-25737",
+  "modified": "2021-05-18T19:14:27Z",
+  "published": "2021-05-18T19:14:27Z",
+  "summary": "Holes in EndpointSlice Validation Enable Host Network Hijack",
+  "details": "A security issue was discovered in Kubernetes where a user may be able to redirect pod traffic to private networks on a Node. Kubernetes already prevents creation of Endpoint IPs in the localhost or link-local range, but the same validation was not performed on EndpointSlice IPs.",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/apiserver"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:N/AC:L/PR:H/UI:N/S:U/C:L/I:N/A:N"
+        }
+      ],
+      "ranges": [
+        {
+          "events": [
+            {
+              "introduced": "1.18.0"
+            },
+            {
+              "last_affected": "1.18.18"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/apiserver"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:N/AC:L/PR:H/UI:N/S:U/C:L/I:N/A:N"
+        }
+      ],
+      "ranges": [
+        {
+          "events": [
+            {
+              "introduced": "1.19.0"
+            },
+            {
+              "last_affected": "1.19.10"
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "k8s.io/apiserver"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:N/AC:L/PR:H/UI:N/S:U/C:L/I:N/A:N"
+        }
+      ],
+      "ranges": [
+        {
+          "events": [
+            {
+              "introduced": "1.20.0"
+            },
+            {
+              "last_affected": "1.20.6"
```

Comment:
```suggestion
              "fixed": "1.20.7"
```


```bash
➜  kubernetes git:(master) git tag --contains 4361731e73d986871d4c9e7374b746475c5b7fad | sort -V | head -n 2
v1.20.7
v1.20.8
```

Reply:
changed in https://github.com/aquasecurity/vuln-list-k8s/pull/86/commits/70a9bc0d10821b99f02e499d2fb8c8acd5080f46